tigerpath/majors_and_certificates/majors/CLA_2018.json

```diff
@@ -0,0 +1,373 @@
+{
+  "type": "Major",
+  "name": "Classics",
+  "code": "CLA",
+  "degree": "AB",
+  "year": 2018,
+  "urls": [
+    "https://ua.princeton.edu/academic-units/department-classics",
+    "https://classics.princeton.edu/programs/undergraduate/tracks"
+  ],
+  "contacts": [
+    {
+      "type": "Chair",
+      "name": "Michael A. Flower",
+      "email": "mflower@princeton.edu"
+    },
+    {
+      "type": "Director of Undergraduate Studies",
+      "name": "Joshua Billings",
+      "email": "jhbillings@princeton.edu"
+    }
+  ],
+  "req_list": [
+    {
+      "name": "Tracks",
+      "max_counted": 1,
+      "min_needed": 1,
+      "explanation": null,
+      "req_list": [
+        {
+          "name": "Classics",
+          "max_counted": 1,
+          "min_needed": "ALL",
+          "explanation": null,
+          "req_list": [
+            {
+              "name": "Prerequisites",
+              "max_counted": 1,
+              "min_needed": 1,
+              "explanation": "To enter this program, a student normally should have completed CLG 108 or LAT 108 or demonstrated intermediate proficiency in Greek or Latin through test scores (SAT, AP), a placement exam, or coursework. A strongly motivated student who has completed CLG 102 or LAT 102 (or CLG 103 or LAT 103) may concentrate, with permission of the departmental representative.",
+              "course_list": [
+                "CLG 108",
+                "LAT 108",
+                "CLG 102",
+                "LAT 102",
+                "CLG 103",
+                "LAT 103"
+              ]
+            },
+            {
+              "name": "Departmentals",
+              "max_counted": 1,
+              "min_needed": "ALL",
+              "explanation": "Eight departmental courses are required. Of these, five must be in the original languages at the 200 level or above, including at least one course at the 300 level. The combination of LAT 104-108, or CLG/LAT 105-108, may be counted as the equivalent of one 200-level course. One course in ancient history (CLA 214, 216, 217, 218, or 219, or HIS 343) must also be included among the departmentals. Students must also take the Junior Seminar in the fall of their junior year.",
+              "req_list": [
+                {
+                  "name": "Original Languages",

```

Comment:
I might need some help with this since I'm a bit confused by the structure you're proposing, though I believe it's probably more accurate than the current structure.